core/src/plugins/filed/BareosFdPluginPostgres.py

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# BAREOS - Backup Archiving REcovery Open Sourced
+#
+# Copyright (C) 2014-2014 Bareos GmbH & Co. KG
+#
+# This program is Free Software; you can redistribute it and/or
+# modify it under the terms of version three of the GNU Affero General Public
+# License as published by the Free Software Foundation, which is
+# listed in the file LICENSE.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.
+#
+# Author: Maik Aussendorf
+#
+# Bareos python plugins class that adds files from a local list to
+# the backup fileset
+
+from bareosfd import *
+from bareos_fd_consts import bJobMessageType, bFileType, bRCs
+import os
+import sys
+import re
+import psycopg2
+import time
+import datetime
+from dateutil import parser
+import dateutil
+import json
+import BareosFdPluginLocalFileset
+from BareosFdPluginBaseclass import *
+
+
+class BareosFdPluginPostgres(
+    BareosFdPluginLocalFileset.BareosFdPluginLocalFileset
+):  # noqa
+    """
+    Simple Bareos-FD-Plugin-Class that parses a file and backups all files
+    listed there Filename is taken from plugin argument 'filename'
+    """
```

Comment:
This comment does not describe the functionality.

Reply:
fixed with latest commit